### DISCUSSION TITLE
Percentages

### DISCUSSION THREAD

**Discussion by gg-g** [marcusolsson/grafana-treemap-panel#27] [Ideas]
Thanks for the plug-in - totally use it.

I wonder if it is worth adding options to show values as percentages in the tile (see 1 in attached picture) and/or in the hover info (2).

![percentages](https://user-images.githubusercontent.com/49113408/135830167-2e51be9d-6f60-4761-99ed-0e1a24fc7f76.png)

**Comment by marcusolsson** [comment]
I think it's a reasonable feature :) Would you mind opening a feature request for it?
